modules/Auth/lib/AuthHandler.cpp

```diff
@@ -124,6 +124,20 @@ TokenHandlingResult AuthHandler::on_token(const ProvidedIdToken& provided_token)
     return result;
 }
 
+void AuthHandler::handle_token_validation_result_update(const ValidationResultUpdate& validation_result_update) {
+    auto connector_id = validation_result_update.connector_id;
+    if (this->evses.find(connector_id) != this->evses.end() and this->evses.at(connector_id)->identifier.has_value()) {
+        EVLOG_info << "Updating validation result on connector: " << connector_id;
+        this->evses.at(connector_id)->identifier->authorization_status =
+            validation_result_update.validation_result.authorization_status;
+        this->evses.at(connector_id)->identifier->parent_id_token =
+            validation_result_update.validation_result.parent_id_token;
```

Comment:
You can maybe add a comment here that other parts of the `ValidationResult` are neglected for now